src/main/java/com/yahoo/bullet/querying/Querier.java

```diff
@@ -301,6 +290,14 @@
     @Getter
     private RunningQuery runningQuery;
 
+    @Getter
+    private Filter filter;
+
+    @Getter
+    private Projection projection;
+
+    private boolean copy;
```

Comment:
Can we not save this and just use the enum? Probably just as fast and looks less cluttered.